addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -13,7 +30,7 @@ options.registry.WebsiteEvent.include({
      */
     allowRoomCreation(previewMode, widgetValue, params) {
         this._rpc({
-            model: this.modelName,
+            model: 'event.event',
```

Comment:
Unless this is a necessary change, would not touch it.

Reply:
This change is necessary because this.modelName no longer exists so this LOC led to another traceback.
Refer [this](https://github.com/odoo/odoo/commit/0aa300785a90655d8eca78a2b26bc724baa2aafa) commit : )